synthtool/gcp/templates/node_library/CONTRIBUTING.md

```diff
@@ -39,6 +39,16 @@ accept your pull requests.
 
 ## Running the tests
 
+1.  To run the tests locally, you will need a Google Cloud Project with a [service account](https://cloud.google.com/docs/authentication/production#cloud-console) and an [API key](https://cloud.google.com/docs/authentication/api-keys)
+
+1.  Set the required environment variables. 
+```
+export GCLOUD_TESTS_PROJECT_ID=YOUR_GCP_PROJECT_ID
```

Comment:
it seems to reset the list in the markdown here when I preview, it might do the right thing if we indent:

```
1.  Set the required environment variables. 
```
export GCLOUD_TESTS_PROJECT_ID=YOUR_GCP_PROJECT_ID
 

 
it seems to reset the list in the markdown here when I preview, it might do the right thing if we indent:

```

```md
1.  Set the required environment variables. 
    \`\`\`
    export GCLOUD_TESTS_KEY=PATH_TO_YOUR_SERVICE_ACCOUNT_KEY.json
    export GCLOUD_TESTS_PROJECT_NUMBER=YOUR_GCP_PROJECT_NUMBER
    export GCLOUD_TESTS_API_KEY=YOUR_API_KEY
   \`\`\`
```

Reply:
oops. Great catch, thank you.